### ISSUE TITLE
 Daylight Saving Time with Timer binding

### ISSUE THREAD

**Issue by MattiasAng** [Azure/azure-functions-host#5837]
Investigative information

Please provide the following:

- Timestamp: 2020-03-29 00:01:00.085
- Function App version: 3
- Invocation ID: 7e6d0ed23f10dc48a03290a414d8bf7b
- Region: West Europe

#### Repro steps

Obviously not so easy to reproduce in Azure environment, but these are the configurations.

1. Set WEBSITE_TIME_ZONE to W. Europe Standard Time
2. Cron schedule on timer binding to 0 1 * * * *

#### Expected behavior

Timer keeps running every hour.

#### Actual behavior

Timer stopped running automatically after switchover to Daylight Saving Time. Invocation ID is the last runtime before it stopped working.

#### Known workarounds

None

#### Related information 

* Programming language used 
Powershell

* Bindings used
Timer

**Comment by heikkilamarko** [comment]
We faced exactly the same problem with several of our timer-triggered functions.

**Comment by fabiocav** [comment]
@brettsam @mathewc I believe this is something we've looked at in the past, correct?

**Comment by brettsam** [comment]
Here's one of the last fixes we made (and it links to some others): https://github.com/Azure/azure-webjobs-sdk-extensions/pull/492

**Comment by fabiocav** [comment]
Yeah, I thought it had been done a while back. I'm wondering if there's an issue with the scale controller not adjusting and activating the app incorrectly, but I would expect that to eventually run, if it was the case.

**Comment by brettsam** [comment]
It is likely just a new bug that we didn't catch previously. If I recall, the other one was caused during fall-back. This one is during spring-forward.

**Comment by fabiocav** [comment]
Will leave it as triaged so we can take a closer look.

**Comment by 23min** [comment]
This is still a problem, nearly a year later.

My v3 app is running in Ireland. The UC is running a timer trigger on Swedish time every day at 17:30.
 I'm setting a CRON as follows: 
`0 30 17 * * *`
WEBSITE_TIME_ZONE is set to `Central European Standard Time` (i.e. GMT+1)

My function triggers at 15:30 instead of 17:30. 

But for a while it has worked fine for a while with these settings. Mystery!
